src/components/download/helperFunctions.js

```diff
@@ -440,22 +440,44 @@ const createBoundingDimensionsAndPositionPanels = (panels, panelLayout, numLines
     panels.tree.width += (spaceBetweenTrees + panels.secondTree.width);
   }
 
-  if (panels.tree && panels.map) {
-    if (panelLayout === "grid") {
-      width = panels.tree.width + padding + panels.map.width;
-      height = Math.max(panels.tree.height, panels.map.height);
-      panels.map.x = panels.tree.width + padding;
+  // Special handling of layout if measurements panel is included
+  // Display as if we are in "full" view to display all filtered measurements
+  if (panels.measurements) {
+    if (panels.tree) {
+      width = Math.max(panels.tree.width, panels.measurements.width);
+      height = panels.tree.height + padding + panels.measurements.height;
+      panels.measurements.y = panels.tree.height + padding;
     } else {
-      width = Math.max(panels.tree.width, panels.map.width);
-      height = panels.tree.height + padding + panels.map.height;
-      panels.map.y = panels.tree.height + padding;
+      width = panels.measurements.width;
+      height = panels.measurements.height;
+    }
+
+    panels.measurementsXAxis.y = height;
+    height += panels.measurementsXAxis.height;
+
+    if (panels.map) {
+      width = Math.max(width, panels.map.width);
+      panels.map.y = height + padding;
+      height += padding + panels.map.height;
+    }
+  } else {
+    if (panels.tree && panels.map) {
+      if (panelLayout === "grid") {
+        width = panels.tree.width + padding + panels.map.width;
+        height = Math.max(panels.tree.height, panels.map.height);
+        panels.map.x = panels.tree.width + padding;
+      } else {
+        width = Math.max(panels.tree.width, panels.map.width);
+        height = panels.tree.height + padding + panels.map.height;
+        panels.map.y = panels.tree.height + padding;
+      }
```

Comment:
In grid mode (with or without a map) the tree & measurements are half-width but on top of each other in the exported SVG. Note that if there are tree, map & measurements panels then the map is full width in grid mode, whereas I think some of these calculations are expecting the tree & map to be side-by-side?

<img width="1679" height="1220" alt="image" src="https://github.com/user-attachments/assets/9e887941-4f4b-488a-b569-e59f7ae1351e" />